File: Vendedor.java
package tp1;

public class Vendedor extends Entidade {
	private int idade;
	private String raca;
	private String nome;
	private String descricao;
	private int id;
	static private int cont = 0;
	
	public Vendedor(String nome, String raca, int idade, String descricao) {
		this.id = ++cont;
		this.nome = nome;
		this.raca = raca;
		this.idade = idade;
		this.descricao = descricao;
	}
	
	public int getIdade() {
		return idade;
	}
	
	public void setIdade(int idade) {
		this.idade = idade;
	}
	
	public String getRaca() {
		return raca;
	}
	public int getId() {
		return id;
	}
	public void setRaca(String raca) {
		this.raca = raca;
	}
	public String getNome() {
		return nome;
	}
	public void setNome(String nome) {
		this.nome = nome;
	}
	public String getDescricao() {
		return descricao;
	}
	public void setDescricao(String descricao) {
		this.descricao = descricao;
	}
}

File: python/Vendedor.py
from Entidade import Entidade

class Vendedor (Entidade):
    __cont = 1
    def __init__(self, nome, raca, idade, descricao):
        self.setId(self.__cont)
        self.setCont(self.__cont+1)
        self.__nome = nome
        self.__raca = raca
        self.__idade = idade
        self.__descricao = descricao
        self.__estoque = []
    @classmethod
    def setCont(cls, cont):
        cls.__cont = cont

    def getEstoque(self):
        return self.__estoque

    def setEstoque(self, estoque):
        self.__estoque = estoque

    def getNome(self):
        return self.__nome

    def setNome(self, nome):
        self.__nome = nome

    def getRaca(self):
        return self.__raca

    def setRaca(self, raca):
        self.__raca = raca

    def getIdade(self):
        return self.__idade

    def setIdade(self, idade):
        self.__idade = idade

    def getDescricao(self):
        return self.__descricao

    def setDescricao(self, descricao):
        self.__descricao = descricao
